pay-api/src/pay_api/services/refund.py

```diff
@@ -219,6 +219,9 @@ def create_routing_slip_refund(cls, routing_slip_number: str, request: Dict[str,
             reason = f"Refund Rejected by {user_name}"
 
         rs_model.status = refund_status
+        rs_model.refund_status = (RoutingSlipRefundStatus.PROCESSED
```

Comment:
I think this should be set to processed in pay-queue when the makes it to `RoutingSlipStatus.REFUND_PROCESSED`

Reply:
I think it should not set PROCESSED directly here, just looking at refund fas flow, it doesn't make sense refund directly without any feedback file received. Need to double check.